src/main/java/org/web3j/spring/autoconfigure/Web3jAutoConfiguration.java

```diff
@@ -61,7 +62,14 @@ private Web3jService buildService(String clientAddress) {
             web3jService = new HttpService(createOkHttpClient());
         } else if (clientAddress.startsWith("http")) {
             web3jService = new HttpService(clientAddress, createOkHttpClient(), false);
-        } else if (System.getProperty("os.name").toLowerCase().startsWith("win")) {
+        } else if (clientAddress.startsWith("wss")) {
```

Comment:
Should be `startsWith("ws")` to support both `ws://` and `wss://`

Reply:
It is true, change is already done.

Reply:
I changed travis configuration build because oraclejdk8 needs a login for download.